app/src/main/java/com/infomaniak/mail/ui/main/folder/ThreadListAdapter.kt

```diff
@@ -85,7 +85,7 @@ class ThreadListAdapter @Inject constructor(
     private val checkMarkSizeOther by lazy { context.resources.getDimension(R.dimen.largeIconSize).toInt() }
 
     private var swipingIsAuthorized: Boolean = true
-    private var displaySeeAllButton = false // TODO: Manage this for intelligent mailbox
+    // private var displaySeeAllButton = false // TODO: Manage this for intelligent mailbox
```

Comment:
I think we should finally suppress this instead of just commenting it

Reply:
Done here: https://github.com/Infomaniak/android-kMail/pull/1774?w=1